ci-operator/config/openshift/release/openshift-release-master__nightly-4.12.yaml

```diff
@@ -612,7 +612,7 @@ tests:
           cpu: 100m
     workflow: ipi-aws
 - as: e2e-telco5g
-  cluster: build01
+  cluster: build05
```

Comment:
This will be changed by our automation. You have to change https://github.com/openshift/release/blob/master/core-services/sanitize-prow-jobs/_config.yaml

/hold

Reply:
Should I leave `ci-operator/jobs/openshift/release/openshift-release-master-periodics.yaml` as it's in this patch? Or to restore it from master.

Reply:
> This will be changed by our automation. You have to change https://github.com/openshift/release/blob/master/core-services/sanitize-prow-jobs/_config.yaml
> 
> /hold

Should I leave `ci-operator/jobs/openshift/release/openshift-release-master-periodics.yaml` as it's in this patch? Or to restore it from master.